website/docs/reference/widgets/table/column-settings.md

```diff
@@ -152,6 +153,38 @@ Based on the Column type, you can edit the cell content. Once done, you can move
 
 <VideoEmbed host="youtube" videoId="Wmitzz4UAGo" title="Editable" caption="Editable"/>
 
+### Column Freeze
+
+Column freeze controls whether the column can be frozen to the left, right or no freeze in the table. Once the column is frozen a Pin icon appears on the respective column in the Columns property.
+
+Below are the scenarios that occur while freezing and unfreezing a column:
+* **Freezing**
+  * A column if frozen to left, get positioned after the last left frozen column.
```

Comment:
Keyur we can rewrite it to say, any new frozen column to the left or right can be seen in the property pane occupying the same index as seen in the table.

Reply:
Yup that works